joint_trajectory_controller/include/joint_trajectory_controller/validate_jtc_parameters.hpp

```diff
@@ -16,12 +16,17 @@
 #define JOINT_TRAJECTORY_CONTROLLER__VALIDATE_JTC_PARAMETERS_HPP_
 
 #include <string>
+#include <vector>
 
 #include "parameter_traits/parameter_traits.hpp"
+#include "rclcpp/parameter.hpp"
+#include "rsl/algorithm.hpp"
+#include "tl_expected/expected.hpp"
```

Comment:
How do we go about these two "new" dependencies? they aren't declared in the package.xml but we get them transitively. A general rule of thumb is if you *explicitly* use code from a dependency, you should declare it as your own, not a transitive one.

Reply:
Agree! I'll add this